<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_language">
       <property name="text">
        <string>Language: </string>
       </property>
       <property name="buddy">
        <cstring>comboBox_language</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_language">
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中文</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_startOfWeek">
       <property name="text">
        <string>Start of Week:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_startOfWeek</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_startOfWeek">
       <item>
        <property name="text">
         <string>Monday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tuesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wednesday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thursday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Friday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Saturday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sunday</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
